contact.html

```diff
@@ -0,0 +1,613 @@
+                                        <div class="button aem-GridColumn aem-GridColumn--default--6">
+                                            <a class="cmp-button" type="button">
+                                                <span class="cmp-button__text">
+                                                    Email Us
+                                                </span>
+                                            </a>
+                                        </div>
+                                        <div class="button aem-GridColumn aem-GridColumn--default--6">
+                                            <a class="cmp-button" type="button">
+                                                <span class="cmp-button__text">
+                                                    Subscribe to Our Newsletter
+                                                </span>
+                                            </a>
+                                        </div>
+                                        <div class="button aem-GridColumn aem-GridColumn--default--6">
+                                            <a class="cmp-button" type="button">
+                                                <span class="cmp-button__text">
+                                                    Shop LeaderImpact Merch                                                </span>
+                                            </a>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="container responsivegrid social aem-GridColumn aem-GridColumn--default--none aem-GridColumn--default--5 aem-GridColumn--offset--default--0 aem-GridColumn--phone--none aem-GridColumn--phone--12 aem-GridColumn--offset--phone--0">
+                                <div class="cmp-container">
+                                    <div class="social-title">
+                                        <div id="title-fca834a3a8" class="cmp-title">
+                                            <h3 class="cmp-title__text">Find Us on Social Media</h3>
+                                        </div>
+                                    </div>
+                                    <div class="container responsivegrid social-logos aem-GridColumn aem-GridColumn--default--none aem-GridColumn--default--12 aem-GridColumn--offset--default--0 aem-GridColumn--phone--none aem-GridColumn--phone--12 aem-GridColumn--offset--phone--0">
+                                        <div class="cmp-container">
+                                            <div class="button aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                                <a id="button-229994a626" class="cmp-button" href="https://pinterest.com">
```

Comment:
@Prystal254 We might need to get a little creative with these icon buttons. I'm not sure what icons AEM has built-in. If we want to use custom SVGs like you've got, we'll need to add those in with pseudo elements and hide the button text. So the HTML will look like a regular text button/anchor, but the CSS will make it look like just an icon. This goes for all three buttons.

Reply:
The core button component has an icon property, which I believe it adds as a class with some prefixes. This is the route I would recommend exploring here.

Reply:
@wrandall22 Is there somewhere we can look at a list of what icons are available? I've never been able to find one.

Reply:
I don't think there are out of the box icons available here. We use a name and write the css to match.

Reply:
@wrandall22 Oh! I didn't realize that's how it worked. I thought there were icons already available and we tapped into them by adding specific identifiers. Good to know.

@Prystal254 So you'll want to use a class like `.cmp-button__icon--icon-pinterest` like you've already done, but we'll need to style it through the CSS rather than custom HTML.